### DISCUSSION TITLE
What does exactly do hyprland_qtutils?

### DISCUSSION THREAD

**Discussion by stefonarch** [hyprwm/Hyprland#8828] [Q&A] [answered]
Running LXQt with Hyprland I removed that package (AUR only) and I can't spot any issues.

![immagine](https://github.com/user-attachments/assets/749e2d9f-3f58-4d2b-ac7c-6ceef95680a6)

Using 
```
misc {
...
    disable_hyprland_qtutils_check = true
...
} 
```
to suppress the warning. Is that more for Qml stuff?

**Comment by stefonarch** [comment]
in the meantime it got a dependency and a package on arch, so this question is hypothetic now.

**Comment by vaxerski** [comment, marked as answer]
It adds a few GUI stuff that hyprland might use at runtime. For example, already shipping with 0.46, is the update screen, which pops up every time you update, reminding you to check the release notes. (so many people don't it's actually annoying!)

In the nearest future, I plan to use it for stuff like ANR, Permission management for screencopy, etc.

So, yes, although with 0.46 you will not have any _major_ problems without qtutils, going forward, you kinda will.

**Reply by stefonarch** [reply]
Thanks.
